File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/responses/CreateOrderResponse.java
package com.elgin.elginexperience.shipay.responses;

import com.google.gson.annotations.SerializedName;

public class CreateOrderResponse {
    @SerializedName("deep_link")
    public String deep_link;

    @SerializedName("order_id")
    public String order_id;

    @SerializedName("pix_dict_key")
    public String pix_dict_key;

    @SerializedName("pix_psp")
    public String pix_psp;

    @SerializedName("qr_code")
    public String qr_code;

    @SerializedName("qr_code_text")
    public String qr_code_text;

    @SerializedName("status")
    public String status;

    @SerializedName("wallet")
    public String wallet;
}

File: Exemplos/App_eXperience_Kotlin/Elgin_AppExperience_Kotlin-font/app/src/main/java/com/example/e1_kotlin_r/shipay/responses/CreateOrderResponse.kt
package com.example.e1_elgin_kotlin.shipay.responses

import com.google.gson.annotations.SerializedName

data class CreateOrderResponse (
    @SerializedName("deep_link")
    var deep_link: String,

    @SerializedName("order_id")
    var order_id: String,

    @SerializedName("pix_dict_key")
    var pix_dict_key: String,

    @SerializedName("pix_psp")
    var pix_psp: String,

    @SerializedName("qr_code")
    var qr_code: String,

    @SerializedName("qr_code_text")
    var qr_code_text: String,

    @SerializedName("status")
    var status: String,

    @SerializedName("wallet")
    var wallet: String
)
